### PR TITLE
Fix chatbot-related code

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Fix chatbot-related code (chatbot server and tools scripts)

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1138

### CODE REVIEW

File: tools/simple-chatbot-server/index.js [hunk 1]
```diff
@@ -23,34 +26,39 @@ app.get('/', (req, res, next) => {
...
     return;
   }
-  const ref = tx.tx_body.operation.ref;
+  const ref = _.get(tx, 'tx_body.operation.ref');
   const parsedRef = CommonUtil.parsePath(ref);
-  const userVal = tx.tx_body.operation.value;
-  if (parsedRef.length !== 6 || parsedRef[0] !== 'apps' || parsedRef[2] !== 'messages' ||
-      parsedRef[5] !== 'user') {
+  const userVal = _.get(tx, 'tx_body.operation.value');
+  if (parsedRef.length !== 6 || parsedRef[0] !== 'apps' || parsedRef[1] !== 'chatbots' ||
```
Comment (woomurf) [inline tools/simple-chatbot-server/index.js:43]: Question) Simple chatbot appId is fixed to "chatbots"?

**Review by woomurf — APPROVED** [review]
LGTM with minor question. Thanks!

File: tools/simple-chatbot-server/index.js (hunk 1 above)
Comment (platfowner) [reply tools/simple-chatbot-server/index.js]: Thanks for asking.
Just parameterized the app name, which should be sync'ed with the tools.

**Comment by platfowner** [comment]
Thanks for the review. Now merging..

**Pull request merged**